ba/exam2/source/js/script.js

```diff
@@ -0,0 +1,334 @@
+      id: "125",
+      alt: "life style",
+      keyword: "life style",
+      url: "./source/img/ls6.png",
+    },
+    {
+      id: "126",
+      alt: "life style",
+      keyword: "life style",
+      url: "./source/img/ls7.png",
+    },
+    {
+      id: "127",
+      alt: "life style",
+      keyword: "life style",
+      url: "./source/img/ls8.png",
+    },
+    {
+      id: "110",
+      alt: "christmas",
+      keyword: "christmas",
+      url: "./source/img/xmas1.jpg",
+    },
+    {
+      id: "111",
+      alt: "christmas",
+      keyword: "christmas",
+      url: "./source/img/xmas2.jpg",
+    },
+    {
+      id: "112",
+      alt: "christmas",
+      keyword: "christmas",
+      url: "./source/img/xmas3.jpg",
+    },
+    {
+      id: "113",
+      alt: "christmas",
+      keyword: "christmas",
+      url: "./source/img/xmas4.jpg",
+    },
+    {
+      id: "114",
+      alt: "christmas",
+      keyword: "christmas",
+      url: "./source/img/xmas5.jpg",
+    },
+    {
+      id: "115",
+      alt: "christmas",
+      keyword: "christmas",
+      url: "./source/img/xmas6.jpg",
+    },
+    {
+      id: "116",
+      alt: "christmas",
+      keyword: "christmas",
+      url: "./source/img/xmas7.jpg",
+    },
+    {
+      id: "117",
+      alt: "christmas",
+      keyword: "christmas",
+      url: "./source/img/xmas8.jpg",
+    },
+    {
+      id: "130",
+      alt: "aircraft",
+      keyword: "aircraft",
+      url: "./source/img/aircraft1.jpg",
+    },
+    {
+      id: "131",
+      alt: "aircraft",
+      keyword: "aircraft",
+      url: "./source/img/aircraft2.jpg",
+    },
+    {
+      id: "132",
+      alt: "aircraft",
+      keyword: "aircraft",
+      url: "./source/img/aircraft3.jpg",
+    },
+    {
+      id: "133",
+      alt: "aircraft",
+      keyword: "aircraft",
+      url: "./source/img/aircraft4.jpg",
+    },
+    {
+      id: "134",
+      alt: "aircraft",
+      keyword: "aircraft",
+      url: "./source/img/aircraft5.jpg",
+    },
+    {
+      id: "135",
+      alt: "aircraft",
+      keyword: "aircraft",
+      url: "./source/img/aircraft6.jpg",
+    },
+    {
+      id: "136",
+      alt: "aircraft",
+      keyword: "aircraft",
+      url: "./source/img/aircraft7.jpg",
+    },
+    {
+      id: "137",
+      alt: "aircraft",
+      keyword: "aircraft",
+      url: "./source/img/aircraft8.jpg",
+    },
+  ];
+
+  class GalleryImage {
+    constructor({ id, alt, keyword, url }) {
+      this.id = id;
+      this.alt = alt;
+      this.keyword = keyword;
+      this.url = url;
+    }
+    render() {
+      let inner = `<div class="gallery__item">
+      <a href="${this.url}" data-fancybox="gallery" data-caption="${this.id}"> 
+      <img class = "gallery__image" src="${this.url}" alt="${this.alt}">
+      </a></div>`;
+      $(".gallery__wrapper").append(inner);
+    }
+  }
+
+  let category = "christmas";
+  let galleryArr = [...gallery];
+  galleryArr = galleryArr.filter((item) => item.keyword == category);
```

Comment:
as we can see you duplicated this code in 327 line of code it means that it should be a separate function for that whole functionality and called 2 times in the appropriate places